src/main/java/org/jvnet/hudson/test/SecuredMockFolder.java

```diff
@@ -69,15 +69,14 @@ public boolean hasPermission(Permission p) {
         return hasPermissionInField(Jenkins.getAuthentication().getName(), p);
     }
     
-    private boolean hasPermissionInField(String sid, @Nonnull Permission p) {
+    private boolean hasPermissionInField(String sid, @NonNull Permission p) {
         if (sid.equals(grantedUser)) {
-            if (grantedPermissions != null && grantedPermissions.contains(p.getId())) {
-                return true;
-            }
+            return grantedPermissions != null && grantedPermissions.contains(p.getId());
         }
         return false;
```

Comment:
Had to read this a couple times to make sure it preserved behavior! Clearer still would be a single

```java
return sid.equals(grantedUser) && grantedPermissions != null && grantedPermissions.contains(p.getId());
```

Reply:
Changed to single line.